### ISSUE TITLE
From torch.Tensor to f-string failed using python3.7, pytorch 1.0.0 and fastai 1.0

### ISSUE THREAD

**Issue by lidh15** [fastai/fastai#1124]
My pytorch version is:
![image](https://user-images.githubusercontent.com/14806954/48303470-7f892a80-e545-11e8-8fbd-cffca2d2ce40.png)
The error is reported:
![image](https://user-images.githubusercontent.com/14806954/48303402-cb879f80-e544-11e8-8e97-674e7c916417.png)
The problem is that in basic_train.py, constructing an f-string with a Tensor failed:
![image](https://user-images.githubusercontent.com/14806954/48303413-e35f2380-e544-11e8-91f9-2e4c526ec3b6.png)
I can reproduce it in python3.7 as following:
![image](https://user-images.githubusercontent.com/14806954/48303437-3933cb80-e545-11e8-81f5-4199bea6caa0.png)
Please let me know if this has been fixed.

**Comment by sgugger** [comment]
What's the loss function you are using? If it returns a rank 0 tensor, you shouldn't have that error.
Look at the result of one of your losses, and it should look like:
`tensor(0.7161, device='cuda:0')` 
not 
`tensor([0.7161], device='cuda:0')`

**Comment by lidh15** [comment]
Thanks for you comment.
Remove the brackets and fixed.
